pkg/apis/cassandraoperator/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -259,7 +259,6 @@ func (in *CassandraDataCenterList) DeepCopyObject() runtime.Object {
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
 func (in *CassandraDataCenterSpec) DeepCopyInto(out *CassandraDataCenterSpec) {
 	*out = *in
-	out.Cluster = in.Cluster
```

Comment:
any reason why this line is removed from the autogen code?

Reply:
`cluster` went from being a `v1.LocalObjectReference` to ` string`, which I guess (no idea how the deep copy stuff works), doesn't require extra handling or something.

I'm still mucking around with how to best represent clusters and multi-DC -- this is a symptom of that.